doc/source/release-notes/Percona-Server-5.7.17-11.rst

```diff
@@ -0,0 +1,94 @@
+all the bug fixes in it, |Percona Server| 5.7.17-11 is the current GA release
+in the |Percona Server| 5.7 series. All of |Percona|'s software is open-source
+and free, all the details of the release can be found in the `5.7.17-11
+milestone at
+Launchpad <https://launchpad.net/percona-server/+milestone/5.7.17-11>`_
+
+New Features
+============
+
+ |Percona Server| has implemented support for per-column ``VARCHAR/BLOB``
+ :ref:`compression <compressed_columns>` for the |XtraDB| storage engine. This
+ also features compression dictionary support, to improve compression ratio for
+ relatively short individual rows, such as JSON data.
+
+ :ref:`innodb_kill_idle_trx` feature has been re-implemented by setting a
+ different socket read timeout value instead of periodically scanning the low
+ |InnoDB| transaction list level. This re-implementation is also addressing
+ some existing bugs: :bug:`1166744`, :bug:`1179136`, :bug:`907719`, and
+ :bug:`1369373`.
+
+Bugs Fixed
+==========
+
+ Logical row for |TokuDB| tables could get inaccurate over time. Bug fixed
+ :bug:`1651844` (:ftbug:`732`).
+
+ Repeated execution of ``SET STATEMENT ... FOR`` could lead to a server crash.
+ Bug fixed :bug:`1392375`.
+
+ ``CREATE TEMPORARY TABLE`` would create a transaction in binary log on read
+ only server. Bug fixed :bug:`1539504`.
+
+ Using :ref:`per_query_variable_statement` with subquery temporary tables could
+ cause a memory leak. Bug fixed :bug:`1635927`.
+
+ Fixed new compilation warnings with GCC 6. Bugs fixed :bug:`1641612` and
+ :bug:`1644183`.
+
+ If a bitmap write I/O errors happened in the background log tracking thread
+ while a ``FLUSH CHANGED_PAGE_BITMAPS`` was executing concurrently could
+ result in a server crash. Bug fixed :bug:`1651656`.
+
+ |TokuDB| was using wrong function to calculate free space in data files. Bug
+ fixed :bug:`1656022` (:tokubug:`1033`).
+
+ ``CONCURRENT_CONNECTIONS`` column in the :table:`USER_STATISTICS` table was
+ showing incorrect values. Bug fixed :bug:`728082`.
+
+ :ref:`audit_log_plugin` when set to ``JSON`` format was not escaping
+ characters properly. Bug fixed :bug:`1548745`.
+
+ |InnoDB| index dives did not detect concurrent tree changes, which couldi
+ return bogus estimates. Bug fixed :bug:`1625151` (upstream :mysqlbug:`84366`).
+
+ :table:`INFORMATION_SCHEMA.INNODB_CHANGED_PAGES` queries would needlessly read
+ potentially incomplete bitmap data past the query range. Bug fixed
+ :bug:`1625466`.
+
+ |Percona Server| ``cmake`` compiler would always check *RocksDB* even if
+ ``-DWITHOUT_ROCKSDB=1`` argument was specified. Bug fixed :bug:`1638455`.
+
+ Lack of free pages in the buffer pool is not diagnosed with
+ :variable:`innodb_empty_free_list_algorithm` set to ``backoff`` (which is the
+ default). Bug fixed :bug:`1657026`.
+
+ ``mysqld_safe`` now limits the use of ``rm`` and ``chown`` to avoid privilege
+ escalation. ``chown`` can now be used only for :file:`/var/log` directory. Bug
+ fixed :bug:`1660265`.
```

Comment:
Thanks to Dawid Golunski (https://legalhackers.com)

This notice is required in 5.5/5.6 too